tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -640,76 +644,100 @@ describe(__filename, () => {
     });
 
-    it('does not render a rating manager without a version', () => {
-      dispatchAddon({ ...fakeAddon, current_version: null });
+    it('renders Rating with add-on ratings', () => {
+      const addon = {
+        ...fakeAddon,
+        ratings: {
+          ...fakeAddon.ratings,
+          average: 4.5,
+        },
+      };
+      dispatchAddon(addon);
       const root = render();
+      const rating = root.find(Rating);
 
-      expect(root.find(RatingManager)).toHaveLength(0);
+      expect(rating).toHaveProp('rating', addon.ratings.average);
     });
 
-    it('handles a submitted review', () => {
-      const addonSlug = 'some-slug';
-      dispatchAddon({ ...fakeAddon, slug: addonSlug });
-      const dispatchSpy = sinon.spy(store, 'dispatch');
-      const root = render({ reviews: null, params: { addonSlug } });
+    it('renders RatingsByStar without an add-on', () => {
+      const root = render({ addon: null });
+      const ratingsByStar = root.find(RatingsByStar);
 
-      const manager = root.find(RatingManager);
-      expect(manager).toHaveProp('onReviewSubmitted');
-      const onReviewSubmitted = manager.prop('onReviewSubmitted');
+      expect(ratingsByStar).toHaveProp('addon', null);
+    });
 
-      // Simulate a review submission.
-      onReviewSubmitted();
+    it('renders RatingsByStar with an add-on', () => {
+      const addon = { ...fakeAddon };
+      dispatchAddon(addon);
+      const root = render();
+      const ratingsByStar = root.find(RatingsByStar);
 
-      sinon.assert.calledWith(dispatchSpy, clearAddonReviews({ addonSlug }));
+      expect(ratingsByStar).toHaveProp('addon');
+      expect(ratingsByStar.prop('addon')).toMatchObject({ id: addon.id });
     });
 
-    it('resets the page after submitting a review', () => {
-      const history = createFakeHistory();
-      const props = {
-        history,
-        location: createFakeLocation({
-          query: { page: 2 },
-        }),
-        params: {
-          addonSlug: fakeAddon.slug,
-        },
-        reviews: null,
-      };
+    it('renders loading text without an add-on', () => {
+      const root = render({ addon: null });
 
-      dispatchAddon({ ...fakeAddon });
+      expect(
+        root.find('.AddonReviewList-addonAverage').find(LoadingText),
+      ).toHaveLength(1);
+    });
+
+    it('renders empty text without add-on ratings', () => {
+      dispatchAddon({ ...fakeAddon, ratings: undefined });
+      const root = render();
 
-      const root = render(props);
+      expect(root.find('.AddonReviewList-addonAverage').text()).toEqual('');
+    });
 
-      const manager = root.find(RatingManager);
-      expect(manager).toHaveProp('onReviewSubmitted');
-      const onReviewSubmitted = manager.prop('onReviewSubmitted');
+    it('renders a fixed star average', () => {
+      dispatchAddon({
+        ...fakeAddon,
+        ratings: {
+          ...fakeAddon.ratings,
+          average: 4.6667,
```

Comment:
Nit: Would it be better to use a `const` for this and in the assertion?

Reply:
Since our test files don't use Flow, I think it's a bit safer to hard code conversions like this. The alternative would be to re-implement the `.toFixed(1)` call but that kind of thing can lead to bitrot without static typing.